tests/python/tank_test/tank_test_base.py

```diff
@@ -218,6 +289,8 @@ def setUp(self, parameters=None):
 
         parameters = parameters or {}
 
+        self._do_io = parameters.get("do_io", True)
```

Comment:
I'll be honest and say I'm not sure why we have a dictionary of parameters in this method. I've went this route because this seems to be the pattern.

I'm assuming it's to stay backwards compatible, altought that doesn't really make sense because you can always have default arguments to methods.